beets/autotag/mb.py

```diff
@@ -276,11 +276,22 @@ def album_info(release):
         disambig.append(release.get('disambiguation'))
     info.albumdisambig = u', '.join(disambig)
 
-    # Release type not always populated.
+    # Retrieves the Release type.
+    # Considers all other release types(including primary and secondary) (new)
+    # and logs them.
     if 'type' in release['release-group']:
         reltype = release['release-group']['type']
         if reltype:
             info.albumtype = reltype.lower()
+    if 'primary-type' in release['release-group']:
+        rel_primarytype = release['release-group']['primary-type']
+        if rel_primarytype:
+            log.debug('Primary Type (new data): ' + rel_primarytype.lower())
+    if 'secondary-type-list' in release['release-group']:
+        if release['release-group']['secondary-type-list']:
+            log.debug('Secondary Type(s) (new data): ' + ', '.join(
+                [secondarytype.lower() for secondarytype in \
```

Comment:
oh wait
So I can remove the backslash and it still accepts it for being in the brackets !

Didn't know that. Oops. Sorry